src/main/java/net/ccbluex/liquidbounce/features/module/Module.kt

```diff
@@ -26,14 +26,18 @@ open class Module @JvmOverloads constructor(
 
     val name: String,
     val category: ModuleCategory,
-    val forcedDescription: String = "",
+    val description: String = translation("module.${name.toLowerCamelCase()}.description"),
```

Comment:
No. This will not work. This will cause the description to not change after the language has changed.